pkg/cloud/services/ssm/ssm-fetch.py

```diff
@@ -0,0 +1,88 @@
+# Copyright 2020 The Kubernetes Authors.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import json
+from cloudinit import url_helper
+from cloudinit import handlers
+from cloudinit import log as logging
+from cloudinit.settings import (PER_INSTANCE)
+from cloudinit import stages
+from cloudinit import util
+
+LOG = logging.getLogger(__name__)
+
+PREFIXES = ["text/cluster-xk8s-io"]
+FILE = "/etc/ssm-userdata.txt"
+
+
+class ClusterXk8sHandler(handlers.Handler):
+
+    def __init__(self, paths, **_kwargs):
```

Comment:
Is the underscore in `_kwargs` typical? I could have sworn I've seen it just as `**kwargs`. I'll also readily admit I am not a Python expert 😄